frontend/src/pages/start/configuration/Configuration.tsx

```diff
@@ -194,6 +197,18 @@ const Configuration: FunctionComponent = () => {
   // TODO(milestone-5): Show back button when editing new project info
   return (
     <div className={classes.root}>
+      <div
+        className={classNames(classes.backButtonWrapper, {
+          [classes.isConfig]: !!id,
+        })}
+      >
+        <IconBackButton
+          onClick={() => {
+            window.history.back();
```

Comment:
Do you think it's better to use `history.back()` from `const history = useHistory()` hook.

See: https://v5.reactrouter.com/web/api/Hooks/usehistory.

Not sure if there are any technical differences between those two, but if nothing less it's for consistency sake.